crates/sparrow-runtime/src/execute/operation.rs

```diff
@@ -175,6 +178,7 @@ impl OperationExecutor {
         max_event_time_tx: tokio::sync::mpsc::UnboundedSender<Timestamp>,
         late_bindings: &EnumMap<LateBoundValue, Option<ScalarValue>>,
         stop_signal_rx: Option<tokio::sync::watch::Receiver<bool>>,
+        udfs: &HashMap<Uuid, Arc<dyn Udf>>,
```

Comment:
Can this be put in the `OperationContext` rather than adding an argument?

Reply:
Done